src/NLog/MessageTemplates/MessageTemplateParameter.cs

```diff
@@ -62,6 +62,28 @@ public struct MessageTemplateParameter
         public readonly string Format;
 
         /// <summary>
+        /// Checks if the <see cref="Format"/> contains reserved letters ('@', '$', 'l')
+        /// </summary>
+        public bool IsReservedFormat
```

Comment:
Another option is to create a static method, so we don't have to construct a new MessageTemplateParameter to check the format

Reply:
It is mostly an informational courtesy for outsiders. No one but Seq, and a few others will be messing with the MessageTemplateParameter

Reply:
And because I have been evil and merged CaptureType into Format

Reply:
Since door was already open with the custom format "l" (lowercase L)

Reply:
OK, but why not a static method? I think that's more clean.

Reply:
Another option is a C# extension, benefit, it "pollutes" the class MessageTemplateParameter less.

Reply:
Both the C# extension and the static-method that takes the struct as parameter will require stack-allocation of the MessageTemplateParameter. Not sure I understand the problem with a struct-property.

Reply:
But we can make it a static-method that takes the format-string as input. Would that be better?

Reply:
Well it's a bit odd here, but the struct reason is a good reason to have it like this.